Dungeoneer/src/com/interrupt/dungeoneer/gfx/GlRenderer.java

```diff
@@ -1524,20 +1524,6 @@ protected void drawUi()
 
 		//drawText("Entities rendered: " + drawnEntities, -camera2D.viewportWidth / 2 + healthSize / 2, -camera2D.viewportHeight / 2 + healthSize / 2, healthSize, Color.WHITE);
 
-		if(game.player.statusEffects != null && game.player.statusEffects.size > 0) {
```

Comment:
So far I've not removed legacy UI. Maybe I should start doing that? I can easy switch branches to compare and would declutter things considerably.

Reply:
Yes, I think that would be a good idea?